SPECS/nano/nano.spec

```diff
@@ -1,28 +1,30 @@
-Summary:	Text editor
-Name:		nano
-Version:	3.0
-Release:        3%{?dist}
-License:	GPLv3+
-URL:		http://www.nano-editor.org/
-Group:		Applications/Editors
-Source0:	http://www.nano-editor.org/dist/v3/%{name}-%{version}.tar.xz
-%define sha1 nano=0396d85df64a7927ca2d3425a0263dacf8502220
+Summary:        Text editor
+Name:           nano
+Version:        6.0
+Release:        1%{?dist}
+License:        GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-BuildRequires:	ncurses-devel
-Requires:	ncurses
+Group:          Applications/Editors
+URL:            https://www.nano-editor.org/
+Source0:        http://www.nano-editor.org/dist/v6/%{name}-%{version}.tar.xz
+%define sha1 nano=0396d85df64a7927ca2d3425a0263dacf8502220
+BuildRequires:  ncurses-devel
+Requires:       ncurses
 
 %description
 The Nano package contains a small, simple text editor
 
 %package lang
-Summary:	Lang for nano
-Requires:	%{name} = %{version}
+Summary:        Lang for nano
+Requires:       %{name} = %{version}
+
 %description lang
 Lang for nano
 
 %prep
-%setup -q -n %{name}-%{version}
+%setup -q
+
 %build
 ./configure --prefix=%{_prefix}      \
```

Comment:
Nit: we can use the `%configure` macro here to get rid of some of the boilerplate configuration (e.g. specification of prefix, sysconfdir)

Reply:
I believe i addressed this correctly but not 100% sure.

Reply:
Looks good, thanks!